pelican/generators.py

```diff
@@ -680,21 +681,9 @@ class StaticGenerator(Generator):
 
     def __init__(self, *args, **kwargs):
         super(StaticGenerator, self).__init__(*args, **kwargs)
+        self.create_symlinks = False
```

Comment:
This is a flag that is set to `True` in `_link_staticfile()` the first time an "Invalid cross-device link" exception is catched. From then on we stop attempting to create hard links and we create soft links instead. Maybe I should rename it `self._cross_device_link_exception_catched`, what do you think?